docs/reference/setup/sysconfig/virtual-memory.asciidoc

```diff
@@ -1,8 +1,8 @@
 [[vm-max-map-count]]
 === Virtual memory
 
-Elasticsearch uses a <<default_fs,`hybrid mmapfs / niofs`>> directory by
-default to store its indices.  The default operating system limits on mmap
+Elasticsearch uses a <<default_fs,`mmapfs`>> directory by
+default for 64bit systems to store its indices.  The default operating system limits on mmap
```

Comment:
In that case, it will be required to change multiple places. The first will be on index-modules-store 
https://www.elastic.co/guide/en/elasticsearch/reference/5.5/index-modules-store.html